flink-python/src/main/java/org/apache/flink/streaming/api/runners/python/beam/BeamPythonFunctionRunner.java

```diff
@@ -250,7 +250,10 @@ public void open(PythonConfig config) throws Exception {
 		Struct pipelineOptions = PipelineOptionsTranslation.toProto(portableOptions);
 
 		if (memoryManager != null && config.isUsingManagedMemory()) {
-			Preconditions.checkArgument(managedMemoryFraction > 0 && managedMemoryFraction <= 1.0);
+			Preconditions.checkArgument(managedMemoryFraction > 0 && managedMemoryFraction <= 1.0,
```

Comment:
How about we ask users to check whether the `consumer-weights` for PYTHON is set to 0, which is wrong because python UDF is used?
Because I think it is the most possible case that the check fails.